### ISSUE TITLE
Summarise the tools in the README

### ISSUE THREAD

**Issue by Llammissar** [google/crush-tools#25]
Right now, I have no idea if any of these tools could be useful to me because the README file links me to a redirect back to github and there's no docs/ dir or anything like that.   Even just a quick summary of what tools are offered and their general use would be very handy.

**Comment by contentfree** [comment]
Yes please… Just came across a recommendation to install the tools, but I have no idea what tools there are!

I did find that if you switch to the `wiki` branch that there's some more info.

**Comment by jimrenwick** [comment]
This code ported from code.google.com years ago, but the docs never made it over. They might still be kicking around somewhere. Let me see if I can dig them up.

**Comment by jimrenwick** [comment]
https://github.com/google/crush-tools/blob/wiki/CrushTutorial.md

https://github.com/google/crush-tools/blob/wiki/CrushAppBasics.md

https://github.com/google/crush-tools/blob/wiki/CheckingForErrors.md

There are other docs + tool docs in the wiki  as well. Check that until someone can get these ported.

Thanks!
